### PR TITLE
7825.master - Views Plugin

### PR DESCRIPTION
**Pull request by ivandudardb**

### CODE REVIEW

File: web/modules/custom/stores/css/map.css
Comment (ivandudardb) [inline web/modules/custom/stores/css/map.css:1]: the file is created for adding styles in the future if necessary. The map does not work correctly if the style file is missing

**Review by HeorhiiBoliuk — APPROVED** [review]
LGTM

File: web/modules/custom/stores/src/Plugin/views/area/PointCustomization.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      '#element_validate' => [
+        [$this, 'validateNumberField'],
+      ],
+    ];
+  }
+
+  /**
+   * Validation for number fields.
+   */
+  public function validateNumberField($element, FormStateInterface $form_state) {
```
Comment (lavrenchuk-dev) [inline web/modules/custom/stores/src/Plugin/views/area/PointCustomization.php:60]: just use #type => number, and you will not need this validation

File: web/modules/custom/stores/css/map.css
Comment (lavrenchuk-dev) [reply web/modules/custom/stores/css/map.css:1]: That can't be the case, this css is not even attached

**Review by lavrenchuk-dev — CHANGES_REQUESTED** [review]

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**